StudyHub/StudyHub/Controllers/OptionController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using StudyHub.BLL.Services.Interfaces.Assignment;
+using StudyHub.Common.DTO.AssignmentTaskOption;
+
+namespace StudyHub.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+[Authorize]
+public class OptionController : Controller
+{
+    private readonly IOptionsService _optionService;
+
+    public OptionController(IOptionsService optionService)
+    {
+        _optionService = optionService;
+    }
+
+    [HttpPost("taskVariantId")]
+    [Authorize(Roles = "Teacher")]
+    public async Task<IActionResult> InsertOption(Guid taskVariantId, List<CreateTaskOptionDTO> taskOptions)
```

Comment:
maybe better create one endpoint for TaskVariants and TaskOptions

Reply:
not exactly, it will be more flexible for teacher to edit or add variants/options